ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -102,7 +103,12 @@ object Cli {
 
     opt[String]("jdbcurl")
       .optional()
-      .text("The JDBC connection URL to a Postgres database containing the username and password as well. If missing the Sandbox will use an in memory store.")
+      .text("This flag is deprecated -- please use --sql-backend-jdbcurl.")
```

Comment:
Please change the SandboxFixture part where this one is still used. Also, please add a test case in `CliSpec`!

Reply:
the `SandboxFixture` uses the config directly. I've amended `CliSpec`.